flashinfer/gemm.py

```diff
@@ -844,8 +844,11 @@ def group_gemm_fp8_nt_groupwise(
 ) -> torch.Tensor:
     from .triton.gemm import compute_padding_mapping
 
-    workspace_buffer = _get_cache_buf(
-        "group_gemm_fp8_nt_groupwise_workspace", 32 * 1024 * 1024, a[0].device
+    int_workspace_buffer = _get_cache_buf(
+        "group_gemm_fp8_nt_groupwise_int_workspace", 32 * 1024 * 1024, a[0].device
```

Comment:
use `a.device` directly, `a[0]` was lagacy of previous version of interface

Reply:
fixed.